apps/website/src/pages/docs/textarea.mdx

```diff
@@ -24,6 +21,36 @@ Text areas (or multi-line fields) are larger text fields that can accommodate mo
 
 It used to be possible to freely resize the text area horizontally in addition to vertically; this pattern is now deprecated, and text areas only allow vertical resizing for better predictability. The resize handle is optional; it may be omitted if resizing of the text area is not desired.
```

Comment:
How long ago was `Textarea` able to be resized horizontally? This part seems unneccessary to me.

Reply:
Removed whole thing